content/docs/1.4.0/troubleshoot/troubleshooting.md

```diff
@@ -1,6 +1,17 @@
 ---
-title: Troubleshooting
-weight: 7
+title: Troubleshooting Problems
+weight: 1
+---
+
+- [Common issues](#common-issues)
+  - [Volume can be attached/detached from UI, but Kubernetes Pod/StatefulSet etc cannot use it](#volume-can-be-attacheddetached-from-ui-but-kubernetes-podstatefulset-etc-cannot-use-it)
+    - [Using with Flexvolume Plugin](#using-with-flexvolume-plugin)
+- [Troubleshooting Guide](#troubleshooting-guide)
+  - [UI](#ui)
+  - [Manager and Engines](#manager-and-engines)
+  - [CSI driver](#csi-driver)
+  - [Flexvolume Driver](#flexvolume-driver)
```

Comment:
We can make each category an individual page in the future. Let's handle this outdated data later.